### PR TITLE
Add call stack print for Error, Fatal and Panic

### PR DESCRIPTION
**Pull request by auxten**
1. Wait for ping success before startDBMS
2. Fix log position error when using WithXXX
3. Make genesis block just one day ago in test mode
4. Add call stack print for Error, Fatal and Panic

### CODE REVIEW

File: cmd/cql-minerd/main.go [hunk 1]
```diff
@@ -248,13 +251,15 @@ func main() {
 				for _, bpNodeID := range bpNodeIDs {
```
Comment (zeqing-guo) [inline cmd/cql-minerd/main.go]: handle timeout

File: cmd/cql-minerd/node.go [hunk 2]
```diff
@@ -73,3 +83,54 @@ func createServer(privateKeyPath, pubKeyStorePath string, masterKey []byte, list
...
+				case <-ch:
+					return
+				case <-time.After(time.Second):
+				}
+
+				err := rpc.PingBP(localNodeInfo, id)
+				if err == nil {
+					ch <- id
+					return
+				} else {
```
Comment (covenantdb) [inline cmd/cql-minerd/node.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
if block ends with a return statement, so drop this else and outdent its block

File: cmd/cql-minerd/node.go [hunk 3]
```diff
@@ -19,7 +19,15 @@ package main
...
 	"os"
+	"strings"
 	"syscall"
+	"time"
+
+	"github.com/CovenantSQL/CovenantSQL/kayak"
+
+	"github.com/pkg/errors"
+
+	"github.com/CovenantSQL/CovenantSQL/proto"
```
Comment (leventeliu) [inline cmd/cql-minerd/node.go]: Remove space lines.

**Review by leventeliu — APPROVED** [review]
LGTM, approve.

**Pull request merged**